CSS/main.css

```diff
@@ -678,6 +762,13 @@ body footer .black .mozilla {
 body footer .black .mozilla .moz {
   margin-left: 40px;
 }
+@media (min-width: 450px) and (max-width: 767px) {
+  body footer .black .mozilla .moz {
+    position: relative;
+    left: 20%;
+    margin-right: 20%;
+  }
```

Comment:
- [ ] **[OPTIONAL] Good job on making the suggested changes by the previous reviewer, 👏 👏**
  However, It would be better to avoid scrolling horizontally on your page as it can be a frustrating experience for users and can create unnecessary barriers to accessing information. To ensure the best user experience 🙏 

- [ ] **[OPTIONAL]** Kindly try to replicate the original design alignments and make sure that your webpage fits the device width as given on the original design.
     
  This will make a good user experience design to provide the best possible user experience. Deviating from the original design can lead to usability issues, confusion, and frustration for users. 😎😎

I have highlighted 👇 some of the alignment issues 

![image](https://user-images.githubusercontent.com/102757126/219636280-80a89bd8-029f-4772-8717-5ba879f45c4c.png)